metadata-ingestion/src/datahub/ingestion/source/snowflake/constants.py

```diff
@@ -36,6 +55,10 @@ class SnowflakeEdition(str, Enum):
     "azure_japaneast": (SnowflakeCloudProvider.AZURE, "japan-east"),
     "azure_southeastasia": (SnowflakeCloudProvider.AZURE, "southeast-asia"),
     "azure_australiaeast": (SnowflakeCloudProvider.AZURE, "australia-east"),
+    "gcp_us_central1": (SnowflakeCloudProvider.GCP, "us-central1"),
+    "gcp_us_east4": (SnowflakeCloudProvider.GCP, "us-east4"),
+    "gcp_europe_west2": (SnowflakeCloudProvider.GCP, "europe-west2"),
+    "gcp_europe_west4": (SnowflakeCloudProvider.GCP, "europe-west4"),
 }
```

Comment:
Let's only add exceptional gov regions here - `aws_us_gov_west_1`, `aws_us_gov_west_1_fhplus`, `aws_us_gov_east_1`

Others will work correctly with default logic - mentioned in comment before `SNOWFLAKE_REGION_CLOUD_REGION_MAPPING`. 

>  `Includes only exceptions to format <provider>_<cloud region with hyphen replaced by _>`